### PR TITLE
VACMS-15891: Update Q&A Content Report View to use Multiple Workflow Filter

### PR DESCRIPTION
**Pull request by chri5tia**
## Description

Relates to #15891 

## Testing done
Locally

## QA steps
Pasted from style guide:
https://prod.cms.va.gov/admin/structure/cm_document/note/124/views-style-guide

As a developer logged in as administrator, go to https://pr15896-p6xmcl1cuiwro2d1zvjakewau8sjznjv.ci.cms.va.gov/admin/structure/views/view/content/edit/page_qa

- [x] Display machine name, not `default`
- [x] URL [`/admin/content/qa`] (unless there are strong feelings to move this to content audit and how)
- [ ] Menu item setting: Is it where it belongs?
- [x] Limit to 25 per page, paginated (style guide says 25, but cypress tests for 25)
- [x] Show result count in header below any exposed filters. (`<p>Displaying @start - @end of @total</p>`)
- [x] Has table caption describing the table (and not changed the content view)
- [ ] Columns
   - [x] Title (sortable)
   - [x] Edit button (not operations, no heading)
   - [ ] Content type (sortable) (if applicable)
   - [ ] Moderation State
   - [ ] Section (field_administration) (sortable)
   - ~~[ ] Is Locked (optional)~~
   - [ ] Last updated (date time and by user) (sortable, default sort, DESC)
   - [ ] [Other content fields as needed] (sortable)
- [ ] Filters
   - [x] Exposed
      - [x] "Title contains" (text search of title field)
      - [x] "Content type" (single value drop down or multivalue? should options be limited) (if applicable)
      - [x] "Moderation state"  - Must use the one for multiple workflows (single value drop down or multivalue?)
      - [x] "Section" (workbench access section; single select, show children)
      - [x] Order of filters matches order of columns
       - [x] Reset button exists
       - [x] Apply button says 'Filter'
- [ ] Tab navigation is alpha sorted or intentionally grouped

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/views.view.content.yml [hunk 1]
```diff
@@ -4,13 +4,11 @@ status: true
 dependencies:
   config:
     - field.storage.node.field_administration
-    - field.storage.node.field_clp_faq_paragraphs
```
Comment (jilladams) [inline config/sync/views.view.content.yml:7]: Contents of the report shouldn't be changing, just workflow, so checking: Was this removal intentional?

File: config/sync/views.view.content.yml [hunk 2]
```diff
@@ -4,13 +4,11 @@ status: true
...
     - field.storage.node.field_facility_location
     - field.storage.node.field_last_saved_by_an_editor
     - field.storage.node.field_other_categories
     - field.storage.node.field_primary_category
     - field.storage.paragraph.field_question
-    - field.storage.paragraph.field_questions
```
Comment (jilladams) [inline config/sync/views.view.content.yml:13]: Same here - was this removal intentional?

File: config/sync/views.view.content.yml (hunk 1 above)
Comment (chri5tia) [reply config/sync/views.view.content.yml:7]: These were added in the last PR but I don't think they're needed. My system is suggesting their removal.

Comment (swirtSJW) [reply config/sync/views.view.content.yml:7]: I agree with Jill.  There are some changes in this PR that seem unexpected.

**Comment by swirtSJW** [comment]
> As a developer logged in as administrator, go to https://va-gov-cms.ddev.site/admin/structure/views/view/content/edit/page_qa

Sending me to my own ddev site is a sure fire way to confuse me :)

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Must:  This caption change got applied to all displays in the View, so we now have things like this
image


I think there are some other issues.. but this one is a definite show stopper.

**Comment by swirtSJW** [comment]
Must:
Tab has gone AWOL
image

I see that you changed the path to be under audit, but it has no tab in that section.

**Comment by swirtSJW** [comment]
I think either the wrong filter is being used for section or it is misconfigured.  It should show the children of a section.
image

**Comment by swirtSJW** [comment]
Style guide-wise  there are some anti-patterns that are being used here.
Admin content is kind of our North star
Admin content is on top, QA audit on the bottom
image

1. We are not supposed to be using the operations field.   Should be a plain edit button.
2. "Updated" should contain the latest revision editor, not have them be in a separate column. 
3. Column order is not in agreement with what the style guide calls out.

**Comment by swirtSJW** [comment]
So it looks like 25  is the new 100.   It looks like when the Outdated content View was created, its pagination got mistakenly or intentionally applied to all the displays in that View.  The good news is, that is not your problem to solve.  :)

**Comment by chri5tia** [comment]
## Caption

Screenshot 2023-10-30 at 3 55 42 PM

The other versions of the content audit view in a [recent, unrelated PR](https://pr15888-uizu61rw6eecymxkelmuvp5ajgfrv6rm.ci.cms.va.gov/admin/structure/views) environment do not have Captions.
Screenshot 2023-10-30 at 3 59 48 PM
So it seems that adding one for Q&A also filled in the one for the template "content". Leaving this empty since we can't cleanly add unique table captions and since the others in this view are empty.

## Menu tab
Right now, the content audit is under administration/content and there doesn't seem to be a better place for it available so I am putting this one back under the content menu parent.

## Everything else
For everything else I copied "All content" and re-created the Q&A view.

File: config/sync/views.view.content.yml (hunk 1 above)
Comment (chri5tia) [reply config/sync/views.view.content.yml:7]: This is evolutionary residue from my earlier work on this and is not needed in the code.

File: config/sync/views.view.content.yml (hunk 2 above)
Comment (chri5tia) [reply config/sync/views.view.content.yml:13]: Not needed.

**Review by omahane — CHANGES_REQUESTED** [review]
###  Menu item setting: Is it where it belongs?
Regarding where this view should live, it should really be in Content Audit tools, making the url
`/admin/content/audit/q&a` and the 
**Local task parent**: "content(content_audit_page)"

### Result count
The QA step says that the result count should be "below any exposed filters", but it is above the filters. This "above the filters" placement also seems consistent with other views. I'm going to read "above" as a typo.

### Columns
- "Content type" is not sortable
- "Updated" comes before "Moderation State"

### Tab navigation
I'm unclear on what this means. If it refers to where the view link shows under Content, it should be moved, as stated above under "Menu item setting"
Screenshot 2023-10-31 at 5 04 49 PM

**Comment by jilladams** [comment]
@swirtSJW @omahane for awareness: I've asked Christia to triage feedback here and identify: 
MUST - what do we need todo now to get this view / PR out of the way of multiple workflows work, and get Christia on to other higher priority sprint goals / sprint work
SHOULD - work we can do as a fast follow

So: not all requested changes may be implemented in this PR, in favor of unblocking sprint priorities. We'll keep track of it. In that spirit: if your feedback isn't critical, please go ahead and approve, and we'll log the notes to take care of in a later PR.

**Comment by jilladams** [comment]
Also for awareness: this view is for CAIA, we won't deliver it to them til the feedback is addressed, so it's low impact to be in a not ideal for a minute while we take care of sprint goals.

**Review by omahane — APPROVED** [review]
I've updated my QA response (https://github.com/department-of-veterans-affairs/va.gov-cms/pull/15896#pullrequestreview-1707306236). Nothing is critical, though I'd say where this views shows in the menu is a SHOULD+. Given that this wasn't an option, I didn't make it such.

**Pull request merged**